### PR TITLE
Toggle menu

### PR DESCRIPTION
**Pull request by Sofinaye**
I added the menu list 

-  and a javascript file that makes the menu list appear when you click the hamburger

- and to disappear when the X button is clicked.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,20 +12,26 @@
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&family=Poppins:wght@400;700&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div id="menu-section" class="menu-section">
+          <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
+        <ul class="menu-list">
+          <li class="menu-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="menu-item"><a href="#about">About</a></li>
+          <li class="menu-item"><a href="#contact">Contact</a></li>
```
Comment (ichala) [inline index.html:20]: - [ ] when I open the mobile menu and click a link to a section, there's a bug preventing me from scrolling or clicking anywhere, kindly fix that 🙏 


bug

File: index.html [hunk 2]
```diff
@@ -12,20 +12,26 @@
...
+        </ul>
```
Comment (ichala) [inline index.html:21]: - [x] as part of the requirements the mobile menu should only be visible in the mobile view , kindly fix the issue where it stays visible for large screens 🙏 

fxxxxxxx

File: index.html [hunk 3]
```diff
@@ -12,20 +12,26 @@
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&family=Poppins:wght@400;700&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div id="menu-section" class="menu-section">
+          <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
```
Comment (ichala) [inline index.html:16]: - [x] kindly increase the size of the "**X**" button and change its color to match the Figma design 

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 


 ## Required Changes ♻️

- [ ] **_🛑 As Part of the requirements , kindly add the [JAVASCRIPT ](https://github.com/microverseinc/linters-config/tree/master/javascript) LINTERS TO YOUR REPO AS WELL 🙏_**
- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Sofinaye, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


CongratulationsGifkaroGIF


## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sofinaye** [comment]
An expandable menu for mobile version

**Pull request merged**